frontend/website/pages/docs/developers/sandbox-node.mdx

```diff
@@ -39,6 +39,14 @@ And stop coda by running.
 docker stop coda
 ```
 
+If you want to execute commands, use the following command to start a shell inside the docker container. 
```

Comment:
I think it might be worth fleshing this out a bit as a source of confusion for some. Just a suggestion:

You can use the [Coda CLI](/docs/cli-reference) to interact with the sandbox node from within the container. The following command opens a shell inside the docker container from where you can issue any of the available [coda commands](/docs/cli-reference).

```
docker exec -it coda bash
```

Reply:
Good suggestion! Changed it.